packages/firestore/test/unit/specs/limit_spec.test.ts

```diff
@@ -362,6 +362,100 @@ describeSpec('Limits:', [], () => {
     }
   );
 
+  /**
+   * Bloom filter:
+   * {
+   *   bits: {
+   *   bitmap: 'Ag=='
+   *   padding: 6
+   *   },
+   *   hashCount: 1
+   *  }
+   * When testing migthContain(), 'collection/a' will return false,
+   * while mightContain('collection/b') will return true.
+   */
+
+  specTest(
```

Comment:
I'm not convinced that this test is necessary. It seems to (a) include a lot of logic that isn't related to existence filters and (b) test the same thing as the other tests. Was there a code path that this test covered that wasn't reached by the other spec tests?

Reply:
Not quite, except for the 'durable-persistence' tag in the spec test. This is the only test that runs with the tag and includes existence filter. 
Will remove this test case for redundancy.

Reply:
I'm pretty sure that the "durable-persistence" tag just means that the test will be skipped if running with memory persistence. So we don't need to worry about "coverage" for "durable-persistence", and this test can be safely deleted.

Reply:
Update the PR's description, which currently mentions "durable-persistence", to no longer mention "durable-persistence".